src/main/java/jp/ac/keio/bio/fun/xitosbml/cui/CLI_Filetype.java

```diff
@@ -0,0 +1,43 @@
+package jp.ac.keio.bio.fun.xitosbml.cui;
+
+import java.io.File;
+import java.util.concurrent.Callable;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "Running XitoSBML-CLI", mixinStandardHelpOptions = true, description = "Saves VIA File option", version = "1.0")
+
+public class CLI_Filetype implements Callable<Integer> {
+
+	@Option(names = "-d", required = true, description = "Directory")
+	String inputDir;
+	@Option(names = "-n", required = true, description = "No.of images to be converted to SBML files")
+	int outputValue;
+	@Option(names = "-i", required = true, description = "Image file id")
+	String imageId;
+	@Option(names = "-o", required = true, description = "Output XML file id")
+	String outputId;
+
+	@Override
+	public Integer call() {
+		String[] input = new String[outputValue];
+		String[] output = new String[outputValue];
+		String dir = inputDir;
+
+		for (int i = 0; i < outputValue; i++) {
+			input[i] = dir + File.separator + imageId + (i + 1) + ".tif";
+			output[i] = dir + File.separator + outputId + (i + 1) + ".xml";
+			System.out.println(input[i]);
+			System.out.println(output[i]);
+			new CuiMainImgSpatial().runCui(input[i], output[i]);
```

Comment:
1. Sure, we could build a string dynamically rather than using input[] and output[]. I will implement this and let you know.
2. I thought it would be better to give the user a choice of how many images they would want to convert. Besides in the Python script as well, we do take some number for the images to be converted. If you so wish, we could find a way to use all the images within the directory.
3. How would you suggest we implement an iterator over a dataset without a pattern for the naming convention? I am sorry for not being able to visualize how to do that.

Reply:
> I thought it would be better to give the user a choice of how many images they would want to convert. Besides in the Python script as well, we do take some number for the images to be converted. If you so wish, we could find a way to use all the images within the directory.

In this snippet, we are making too many assumptions. The file extension is ".tif", the file name is in a specific pattern such as image1, image2... , the numbering convention (maybe it can be zero-padded). 
If we just look into the given directory, we won't encounter this problem. Also, if the user wanted to control how many images they want to run, they can just split it into subdirectories   


> How would you suggest we implement an iterator over a dataset without a pattern for the naming convention? I am sorry for not being able to visualize how to do that.

Determine if the input is an image or a directory.
If it's a directory,  look in the directory for all images (files with extensions .tif, .tiff etc.).